src/api/.rubocop.yml

```diff
@@ -65,6 +65,14 @@ Style/HashSyntax:
   Exclude:
     - 'config/routes.rb'
 
+# Checks the indentation of the first element in an array
+# Configuration parameters: EnforcedStyle, SupportedStyles, IndentationWidth.
+# SupportedStyles: special_inside_parentheses, consistent, align_brackets
+Style/IndentArray:
+  Enabled: true
+  EnforcedStyle: consistent
```

Comment:
Talking about styling I always prefer the 'standard' if there is not a good reason for changing. And in this case I personally prefer the default too, I think it is more readable. @openSUSE/open-build-service what others think? That would be the difference:

``` Ruby
# defaul
array = [
  :value
]
but_in_a_method_call([
                       :its_like_this
                     ])
# consistent
array = [
  :value
]
and_in_a_method_call([
  :no_difference
])
```